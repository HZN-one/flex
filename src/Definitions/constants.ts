export const DUMMY_IMAGE_URL =
  "https://dummyimage.com/250x180/e5e5e5/e5e5e5.jpg";

export const DUMMY_COMPANY_IMAGE_URL =
  "https://www.freepnglogos.com/uploads/netflix-logo-circle-png-5.png";
