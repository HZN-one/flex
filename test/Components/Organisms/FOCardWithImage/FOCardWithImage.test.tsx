/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

import { FOCardWithImage } from "@Organisms";
import { DUMMY_IMAGE_URL } from "@Definitions/constants";

describe("FOCardWithImage", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FOCardWithImage
        testID="card-example"
        image={<img src={DUMMY_IMAGE_URL} />}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
