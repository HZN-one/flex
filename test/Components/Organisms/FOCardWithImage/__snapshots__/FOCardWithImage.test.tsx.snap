// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FOCardWithImage should match snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
    data-testid="card-image-example"
  >
    <div
      class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    >
      <div
        class="MuiBox-root css-1dzqdzx"
      >
        <img
          src="https://dummyimage.com/250x180/e5e5e5/e5e5e5.jpg"
        />
      </div>
      <div
        class="MuiBox-root css-l7tphn"
      >
        <div
          class="MuiCardHeader-root css-185gdzj-MuiCardHeader-root"
        >
          <div
            class="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
            >
              Card Image Title
            </span>
            <span
              class="MuiTypography-root MuiTypography-body1 MuiCardHeader-subheader css-nrdprl-MuiTypography-root"
            >
              Card Image Subheader
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
